<launch>
  
  <arg name="config_path" default="./config.json"/>
  <arg name="tcp_port" default="10000"/>
  <arg name="max_clients" default="10"/>

  <node name="visus_bridge" pkg="visus_tcp_bridge" type="main.py" respawn="false" output="screen">
    <param name="config_path" type="string" value="$(arg config_path)"/>
    <param name="tcp_port" type="int" value="$(arg tcp_port)"/>
    <param name="max_clients" type="int" value="$(arg max_clients)"/>
  </node>

</launch>
